pgoapi/rpc_api.py

```diff
@@ -53,10 +53,10 @@
 
 class RpcApi:
 
-    RPC_ID = 0
-    START_TIME = 0
-
     def __init__(self, auth_provider, device_info):
+        self.RPC_REQUEST = 1
```

Comment:
Correct, and there are other remaining inconsistencies with the other APIs.

Closing my pgoapi fork, in favor of RM taking over pgoapi to maintain it, considering pgoapi's RPC_ID RNG has been outdated for over 7 months.